src/Uno.UI.RuntimeTests/Uno.UI.RuntimeTests.csproj

```diff
@@ -65,7 +65,7 @@
 
 	<ItemGroup Condition="'$(TargetFramework)'=='uap10.0.17763'">
 		<PackageReference Include="System.Numerics.Vectors" Version="4.3.0" />
-		<PackageReference Include="Microsoft.UI.Xaml" Version="2.5.0" />
+		<PackageReference Include="Microsoft.UI.Xaml" Version="2.7.0-prerelease.210816001" />
```

Comment:
IIRC, @MartinZikmund pointed that this can't be updated alone since it will require a version higher than `uap10.0.17763`.

Reply:
Good point... it seems to work however. @MartinZikmund do you know if this might break something?

Reply:
Interesting, when I did this it complained that the target needs to be higher 🤔 . But if it compiles ok for you, maybe something changed 😀